### PR TITLE
feat: add support for Chrome Beta

### PR DESCRIPTION
**Pull request by sanchay9**
I know there should be a better way to do this?

### CODE REVIEW

File: lua/telescope/_extensions/bookmarks/chrome_beta.lua [hunk 1]
```diff
@@ -0,0 +1,13 @@
+local chrome = require "telescope._extensions.bookmarks.chrome"
+
+local chrome_beta = {}
+
+---Collect all the bookmarks for the Brave browser.
+---NOTE: Brave and Google Chrome uses the same underlying format to store bookmarks.
```
Comment (dhruvmanila) [inline lua/telescope/_extensions/bookmarks/chrome_beta.lua:6]:
```suggestion
---Collect all the bookmarks for Google Chrome Beta.
```

File: lua/telescope/_extensions/bookmarks.lua [hunk 2]
```diff
@@ -25,6 +25,7 @@ local title = {
   brave = "Brave",
   buku = "Buku",
   chrome = "Chrome",
+  chrome_beta = "Chrome",
```
Comment (dhruvmanila) [inline lua/telescope/_extensions/bookmarks.lua:28]:
```suggestion
  chrome_beta = "Chrome (beta)",
```

**Review by dhruvmanila — CHANGES_REQUESTED** [review]
> I know there should be a better way to do this?

Well, right now this is the only way lol 😉 

A few small changes and this will be good to go.

File: lua/telescope/_extensions/bookmarks.lua (hunk 2 above)
Comment (sanchay9) [reply lua/telescope/_extensions/bookmarks.lua:28]: I think I would prefer the prompt title to be just Chrome 😄

**Review by dhruvmanila — APPROVED** [review]

**Comment by dhruvmanila** [comment]
Thanks!

**Pull request merged**